src/components/ExportDialog/ExportDialog.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
```suggestion
import React from 'react';
```
These were unused. Removing them will get rid of Typescript error.